scripts/get-from-collector.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-partnerName="$1"
-decoded_password="$2"
-endpoint="http://localhost:8080"
+endpoint="$1"
+partnerName="$2"
+decoded_password="$3"
```

Comment:
Add a safety check here to make sure these are populated.

Reply:
you mean make sure they're not empty?

Reply:
Same thing 😄